### DISCUSSION TITLE
What does "Compact Realm" do under "Memory"

### DISCUSSION THREAD

**Discussion by Modsnotworking** [ppy/osu#16583] [Q&A] [answered]
So "Memory" has two options _now_. I am just curious to know what does "Compact Realm" do??

**Comment by peppy** [comment, marked as answer]
It's a debug option so not really relevant if you don't understand it.

You can use google if you're curious and want to learn more.

https://www.google.com/search?client=safari&rls=en&q=compact+realm&ie=UTF-8&oe=UTF-8
